### ISSUE TITLE
Some error while setup

### ISSUE THREAD

**Issue by Valonix** [linnovate/mean#1605]
Hello, after command init i have this error

Do you want to set up an admin user?
? Yes y
? Please provide your email so we can create your first admin user email@gmail.com

/usr/lib/node_modules/mean-cli/node_modules/rx/dist/rx.js:77
    throw e;
    ^
true

What is this?

**Comment by liorkesos** [comment]
I have recreated the problem.
Will check it out.
Lior

On Mon, Jul 4, 2016 at 4:23 PM, Valeriy Donika <notifications@github.com>
wrote:

> Hello, after command init i have this error
>
> Do you want to set up an admin user?
> ? Yes y
> ? Please provide your email so we can create your first admin user
> email@gmail.com
>
> /usr/lib/node_modules/mean-cli/node_modules/rx/dist/rx.js:77
> throw e;
> ^
> true
>
> What is this?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/linnovate/mean/issues/1605>, or mute the thread
> <https://github.com/notifications/unsubscribe/AAO9I4FxUg-akqD8L18scT9u5-HyuAArks5qSQlAgaJpZM4JEXeo>
> .
>



-- 

Lior Kesos - http://www.linnovate.net
Linnovate - Community Infrastructure Care
mail: lior@linnovate.net
office: +972 722500881
cell: +972 524305252
skype: liorkesos

**Comment by Valonix** [comment]
I am first try this framework.
I created droplet on DO ubuntu 14.04, 
Add user and give them sudo
Login by this user
Installed node latest
Updated npm
installed mongodb (it is work)
Installed gulp-globally and bower
Installed sudo npm install -g mean-cli 
Then when i tried init i have error.
After i tried cd project folder and npm install and nothing...no result about successfully installing modules =(


My configurations steps is correct?
